Classes/common/CDTFetchChanges.h

```diff
@@ -67,6 +67,28 @@
  */
 @property (nonatomic, copy) NSString *startSequenceValue;
 
+/**
+ Use this property to limit the total number of results you want to get.
+ 
+ By default, its value 0 which means that it will returns as many as possible.
+ 
+ Set it before executing the operation or submitting it to a queue.
+ */
+@property(nonatomic, assign) NSUInteger resultsLimit;
```

Comment:
Do you refer to this [line](https://github.com/cloudant/CDTDatastore/pull/177/files#diff-bac15dfde454859cd8e3a8dc04fcf29fR72)? It is not ignored, `CDTFetchChanges` gets the changes in the datastore in groups of 500 records or less if `resultsLimit` is smaller than that. But if `resultsLimit` is bigger than 500, `CDTFetchChanges` will get the changes in groups of 500 records until it reaches the limit.